package.json

```diff
@@ -70,7 +70,8 @@
     "typescript": "^4.0.2",
     "typescript-eslint-language-service": "^4.0.1",
     "vitepress": "^0.7.0",
-    "vue": "^3.0.0"
+    "vue": "^3.0.0",
+    "yarn": "^1.22.10"
```

Comment:
please don't include to devDependencies. 🙏 
package manager should install globally your enviroment!